modules/runtime/pages/fault-tolerance.adoc

```diff
@@ -23,6 +23,7 @@ analyses the exception and determine if the work should be retried automatically
 
 When the exception is retryable, the platform queues the work again, with a delay. That can be done up to 10 times
 with the delay increasing gradually from 1 second to more than 1 hour by default.
+The exact delay is randomized, to avoid repeated congestion situations (a typical case of those are database deadlocks, when retrying all the failed works at the same time will lead to more deadlocks).
```

Comment:
Add dump carriage return:
```suggestion
with the delay increasing gradually from 1 second to more than 1 hour by default. +
The exact delay is randomized, to avoid repeated congestion situations (a typical case of those are database deadlocks, when retrying all the failed works at the same time will lead to more deadlocks).
```